src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&family=Sofia&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img class="card__image" src="./images/imac.jpeg" 
+            alt="Product Image">
+
+      <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="card__code">Product code: 195434</div>
+      
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--noactive"></div>
```

Comment:
don't use active/noactive classes, just copy this block and styles from stars task. You should have ability to change count of active stars changing only one modifier